test/integration/golden_response/get_related_locations/county.json

```diff
@@ -1,18 +1 @@
-{
-  "Count_Person": {
-    "rankFromTop": 18,
-    "rankFromBottom": 3172
-  },
-  "Median_Age_Person": {
-    "rankFromTop": 2583,
-    "rankFromBottom": 603
-  },
-  "Median_Income_Person": {
-    "rankFromTop": 22,
-    "rankFromBottom": 3164
-  },
-  "UnemploymentRate_Person": {
-    "rankFromTop": 1691,
-    "rankFromBottom": 1495
-  }
-}
\ No newline at end of file
+{}
```

Comment:
The data is missing here. @pradh @Spaceenter

Reply:
Also potentially from rankings pipeline migration.

Reply:
This should be fixed by cl/379804788.